packages/flutter/test/widgets/physical_model_test.dart

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:io';
```

Comment:
This is a pretty big import. Might be best to just `import 'dart:io' show Platform;`

Reply:
Good point. Just pushed a fix for this.